release/src/main/scripts/verify_release_build.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# This script will run pre-installations and run release build.
+
+set -e
+
+GIT_REPO_URL=https://github.com/apache/beam.git
+LOCAL_CLONE_DIR=release_build
+BEAM_ROOT_DIR=beam
+
+echo "Which branch you want to verify release build: "
+read branch
+
+echo "=====================Environment Variables====================="
+echo "working branch: ${branch}"
+echo "local repo dir: ~/${LOCAL_CLONE_DIR}/${BEAM_ROOT_DIR}"
+echo "==============================================================="
+
+echo "====================Starting Pre-installation=================="
+cd ~
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+sudo python get-pip.py
```

Comment:
My concern is that we ask for root permissions and install software without saying anything to the user of this script.

How about:
- We first check whether pip etc. is installed or not.
- If installed, we just use them.
- If not installed we ask the user, if the user accepts then we install it. Otherwise it fails here,

Reply:
Agree. Will fix in next commit. Thanks for suggestion!

Reply:
Fixed in commit https://github.com/apache/beam/pull/5973/commits/ef08ee9e0b4dcb57ff3f154a5bc929b98a444234